### DISCUSSION TITLE
Nginx unable to find livewire.js from vendor folder

### DISCUSSION THREAD

**Discussion by csandreas1** [livewire/livewire#8192] [1. Help] [answered]
I've set up a Docker volume to map the public folder with Nginx, but I'm encountering a 404 error when accessing livewire.js, which is located inside the vendor folder.

How can I properly configure Nginx to serve livewire.js without encountering the 404 error?

![image](https://github.com/livewire/livewire/assets/21238024/a60d52f7-6bc1-4945-ab86-0483f475b986)

Nginx conf:

```
server {
    listen 8060;
    server_name localhost;

    root /var/www/html/public;
    index index.php index.html index.htm;

    location ~* \.(jpg|jpeg|gif|css|png|js|ico|html)$ {
        try_files $uri $uri/ =404;
        access_log off;
        expires max;
    }

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }

    location ~ \.php$ {
        include fastcgi_params;
        fastcgi_pass app:9000;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
        fastcgi_param PATH_INFO $fastcgi_path_info;
        fastcgi_param PHP_VALUE "date.timezone=UTC";
    }

    location ~ /\.(?!well-known).* {
        deny all;
    }

    error_log  /var/log/nginx/error.log;
    access_log /var/log/nginx/access.log;
}
```

**Comment by danie-ramdhani** [comment, marked as answer]
maybe this helpful: https://livewire.laravel.com/docs/installation
(scroll to this position)
![image](https://github.com/livewire/livewire/assets/58886605/08a88dbd-1189-4cca-9c17-241735e3b42d)

**Comment by csandreas1** [comment]
thanks @danie-ramdhani adding the below in nginx configuration worked

```
location = /livewire/livewire.js {
    expires off;
    try_files $uri $uri/ /index.php?$query_string;
}
```
